test/libsolidity/SemanticTest.cpp

```diff
@@ -467,3 +468,16 @@ std::optional<bytes> SemanticTest::builtinSmokeTest(FunctionCall const& call)
 	}
 	return result;
 }
+
+std::optional<bytes> SemanticTest::builtinContractBalance(FunctionCall const& call)
+{
+	soltestAssert(call.arguments.parameters.size() <= 1, "");
+	if (call.arguments.parameters.empty())
```

Comment:
I think this function can be greatly simplified:
```
h160 address = call.arguments.parameters.empty() ? m_contractAddress : h160(call.arguments.parameters.at(0).rawString);
return util::toBigEndian(SolidityExecutionFramework::balanceAt(address);
```

Reply:
Nice :)